### PR TITLE
Update from original

### PR DESCRIPTION
**Pull request by TommyHubbard**

### CODE REVIEW

File: runestone/assess/js/timed.js [hunk 1]
```diff
@@ -486,6 +503,21 @@ Timed.prototype.tookTimedExam = function () {
 };
 
 Timed.prototype.finishAssessment = function () {
+<<<<<<< HEAD
```
Comment (bnmnetp) [inline runestone/assess/js/timed.js]: You  did not resolve the conflicts fully.  see here and line 520 and 534

Comment (TommyHubbard) [inline runestone/assess/js/timed.js]: Hi - I've resolved the conflicts and would like to try again.


Thomas Hubbard




Date: Sun, 11 Oct 2015 14:17:29 -0700

**Comment by bnmnetp** [comment]
So there are a bunch of files that got added to the repository that should not be there:

* timed_BACKUP_15132.js 
* timed_REMOTE_15132.js
* timed_BASE_15132.js
* timed_LOCAL_15132.js

I suspect you accidentally added them with a git add.

If you confirm that I can get rid of these I will ``git rm`` them on my test branch and then accept the PR.

Brad

**Comment by TommyHubbard** [comment]
Hi Brad - Yes, those can be removed.   You're right, I used git add. and added everything. I believe those files came from the first conflict I encountered.   I will be more mindful with future pull requests.

Thanks,

Tommy‎ Hubbard
Sent from my BlackBerry 10 smartphone.
From: Brad Miller
Sent: Monday, October 12, 2015 6:53 PM
To: RunestoneInteractive/RunestoneComponents
Reply To: RunestoneInteractive/RunestoneComponents
Cc: Tommy Hubbard
Subject: Re: [RunestoneComponents] Update from original (#98)


So there are a bunch of files that got added to the repository that should not be there:

* timed_BACKUP_15132.js
* timed_REMOTE_15132.js
* timed_BASE_15132.js
* timed_LOCAL_15132.js

I suspect you accidentally added them with a git add.

If you confirm that I can get rid of these I will ``git rm`` them on my test branch and then accept the PR.

Brad

---

**Pull request merged**